sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -210,41 +219,67 @@ export class ManagementClient extends LinkEntity {
    */
   async getPartitionProperties(
     partitionId: string,
-    options?: { retryOptions?: RetryOptions; abortSignal?: AbortSignalLike }
+    options: OperationOptions & { retryOptions?: RetryOptions } = {}
   ): Promise<PartitionProperties> {
-    if (!options) {
-      options = {};
-    }
-    const securityToken = await this.getSecurityToken();
-    const request: Message = {
-      body: Buffer.from(JSON.stringify([])),
-      message_id: uuid(),
-      reply_to: this.replyTo,
-      application_properties: {
-        operation: Constants.readOperation,
-        name: this.entityPath as string,
-        type: `${Constants.vendorString}:${Constants.partition}`,
-        partition: `${partitionId}`,
-        security_token: securityToken?.token
-      }
-    };
+    throwErrorIfConnectionClosed(this._context);
+    throwTypeErrorIfParameterMissing(
+      this._context.connectionId,
+      "getPartitionProperties",
+      "partitionId",
+      partitionId
+    );
+    partitionId = String(partitionId);
+
+    const clientSpan = this._createClientSpan(
```

Comment:
This brings up something I didn't notice before. I think we should probably be creating the span before doing the error checks, that way these errors would show up in a trace. Either you or I can log an issue to look into creating the span prior to throwing an error in this and other APIs.

Reply:
Fair point. I would say that we should review all cases where we use tracing to see if this pattern occurs elsewhere. Can you create an issue please?

Reply:
Created #9536 9536